javaparser-core/src/main/java/com/github/javaparser/printer/concretesyntaxmodel/CsmTextBlock.java

```diff
@@ -34,7 +35,8 @@ public CsmTextBlock(ObservableProperty property) {
 
     @Override
     public void prettyPrint(Node node, SourcePrinter printer) {
-        printer.print("\"\"\"\n");
+        LineEnding lineEnding = node.getLineEndingStyleOrDefault(LineEnding.SYSTEM);
+        printer.print("\"\"\"" + lineEnding.toString());
```

Comment:
As highlighted by @matozoid , textblocks are picky about line endings 

https://github.com/javaparser/javaparser/pull/2685#issuecomment-631008605

https://openjdk.java.net/jeps/378#1--Line-terminators

Reply:
Hmm, It has just occurred to me (in the couple mins since posting my reply) that maybe the **VALUE** is always interpreted as \n, but it is unclear to me whether the **source code** within a java file can have different line endings.

Regardless, it needs some more attention +- a comment to explain.

Reply:
That should definitely remain "\n", textblocks never generate "\r\n" or so.

Reply:
Ah yes, just realised that at this point we're it the CSM dealing with interpreted values (as opposed to source code) so my last comment isn't applicable.

Reply:
And the change we're discussing breaks text blocks, because now they generate platform specific EOL's, which they should not do :-)

Reply:
Reverted this and added a line comment to explain